scripts/setupJest.js

```diff
@@ -1,4 +1,5 @@
 import '@testing-library/jest-dom/extend-expect.js';
+import 'regenerator-runtime/runtime';
```

Comment:
Can you add a comment about what problem this solves?

Reply:
DONE